configure.ac

```diff
@@ -251,10 +251,12 @@ if test "$ac_python_ldflags_set_by_user" != "true"; then
     PYTHON_LDFLAGS="-l$ac_python_library"
   fi
 fi
+AS_VAR_SET(fluxpyexecdir, $pyexecdir/flux)
+AC_SUBST(fluxpydir)
```

Comment:
should this be `fluxpyexecdir`?

Reply:
Good catch! Actually that is a rebase error and those lines need to be removed (`fluxpyexecdir` was from an earlier version of the PR)